### DISCUSSION TITLE
How to properly update a game shared between Linux and Windows?

### DISCUSSION THREAD

**Discussion by guihkx** [legendary-gl/legendary#453] [Q&A] [answered]
I have Rocket League installed on my external hard drive (it's NTFS). I also have Windows and Linux on dual boot, so sometimes I end up playing on either.

My question is this one: If I update Rocket League on Linux (`legendary update Sugar`) and then boot into Windows and try to launch the game there, Legendary will ask me to update the game again.

What command should I run now? If I run `legendary update Sugar`, it seems a bit redundant to me:

- It wastes internet bandwidth 
- Legendary will overwrite files that I already have

The logical thing to me, would be running `legendary verify Sugar`, however, I'm not sure if Legendary will compare file hashes against the latest update (that it thinks I don't have), or will just compare hashes against the outdated version it thinks I have.

If it's the first option, then that's exactly what I want! If it's the latter, then I assume I'll still need to run `legendary update Sugar`, no?

So, what would be the most logical thing to do here?

Any help is appreciated!

**Comment by derrod** [comment, marked as answer]
`legendary repair Sugar --repair-and-update` should work, but will still go through the entire verification process. There isn't really a better way.

**Reply by guihkx** [reply]
Fair enough. I'm okay with that :)

Thanks!
